src/averageAges.js

```diff
@@ -37,7 +42,16 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = withChildren
```

Comment:
Same suggestion about procedural approach.
1. Filter out all women because we need it anyway
2. Then, use additional "withChildren" filter if expected